### ISSUE TITLE
Shortcut flip-flop between Create Element and Annotation Tool

### ISSUE THREAD

**Issue by eshattow** [IfcOpenShell/IfcOpenShell#4694]
Blender 4.1.1 from blender.org and BlenderBIM from https://github.com/IfcOpenShell/IfcOpenShell/releases/download/blenderbim-240518/blenderbim-240518-py311-linux.zip

Reproduce strategy:

Hit a few times this key sequence shortcut: `ctrl+spacebar``ctrl+numpad 6`

The activated tool flip-flips between Create Element and Annotation Tool

I would guess this should not be like that? Or are shortcuts not well-defined yet and always dynamic?

**Comment by sboddy** [comment]
Yeah, confirmed this still happens and causes shortcuts to be unpredictable.

**Comment by BIMvoice** [comment]
Traced this. Neither "Create Element" (`bim.bim_tool`) nor "Annotation" (`bim.annotation_tool`) has an explicit activation shortcut, so Blender's toolbar popup (Shift+Space) auto-assigns them number keys by their position in the toolbar. That position shifts as the active tool changes, so the same key keeps flip-flopping between the two tools, hence the unpredictability.

Fix in #8349: give each tool its own stable key in the "Toolbar Popup" keymap (the same place Blender's builtin tools get fixed letters), Create Element = `C`, Annotation = `A`. Each key now deterministically activates one tool.
